src/main/java/com/team4/testingsystem/controllers/UsersController.java

```diff
@@ -17,23 +19,36 @@
 @RestController
 public class UsersController {
     private final UsersService usersService;
+    private final TestsService testsService;
 
     @Autowired
-    public UsersController(UsersService usersService) {
+    public UsersController(UsersService usersService, TestsService testsService) {
         this.usersService = usersService;
+        this.testsService = testsService;
     }
 
     @ApiOperation("Get list of all coaches in the system")
     @GetMapping("/coaches")
     public List<UserDTO> getCoaches() {
-        return usersService.getUsersByRole(Role.COACH).stream()
-                .map(UserDTO::new)
-                .collect(Collectors.toList());
+        return convertToDTO(usersService.getUsersByRole(Role.COACH));
+    }
+
+    @GetMapping("/employees")
+    public List<UserDTO> getAllUsers() {
+        List<UserDTO> users = convertToDTO(usersService.getAll());
+        testsService.attachAssignedTests(users);
```

Comment:
please, move service logic outside from controller

Reply:
If I move attaching tests to service, than DTO will be modified on a service layer...

Reply:
BTW, why attaching assigned tests to DTO can't be done in the controller?

Reply:
Service should work with converting DTO`s to object and back. All the other logic should be made on another layer.

So, the service should make all the logic on itself and return only the result with users, that have (or don't have) corresponging tests